bedrock/firefox/templates/firefox/whatsnew/index.html

```diff
@@ -20,41 +23,28 @@
   {{ css_bundle('firefox_whatsnew') }}
 {% endblock %}
 
-{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
-
-{% set send_to_device_message_set = 'fx-whatsnew' %}
+{% set android_url = play_store_url('firefox', 'firefox-whatsnew') %}
+{% set ios_url = app_store_url('firefox', 'firefox-whatsnew') %}
 
 {% block wnp_content %}
-  <section class="wnp-content-main{% if show_send_to_device %} show-send-to-device{% endif %}">
-    <div class="mzp-l-content mzp-t-content-md">
-      <header>
-        <div class="mzp-c-logo mzp-t-logo-lg mzp-t-product-firefox"></div>
-      {# If user is in a locale with translated basket messages... #}
-      {% if show_send_to_device %}
-        {% if ftl_has_messages('whatsnew-s2d-want-privacy-on-every-device', 'whatsnew-s2d-you-got-it-get-firefox-for') %}
-          <h1 class="wnp-main-title">{{ ftl('whatsnew-s2d-want-privacy-on-every-device') }}</h1>
-          <p class="wnp-main-tagline">{{ ftl('whatsnew-s2d-you-got-it-get-firefox-for') }}</p>
-        {% else %}
-          <h1 class="wnp-main-title">{{ ftl('whatsnew-s2d-send-firefox-to-your-phone') }}</h1>
-        {% endif %}
-      {# For users not in a locale with translated basket messages... #}
-      {% else %}
-        <h1 class="wnp-main-title">{{ ftl('whatsnew-s2d-download-firefox-for-your') }}</h1>
-      {% endif %}
-      </header>
-
-      {# Only certain locales get send_to_device, else they see a QRCode. #}
-      {% if show_send_to_device %}
-        <div id="send-to-device-wrapper" class="primary-cta">
-          {{ send_to_device(include_title=False, message_set=send_to_device_message_set, spinner_color='#fff;') }}
-        </div>
-      {% else %}
-        <div id="qr-wrapper" class="primary-cta">
-          <img src="{{ static('img/firefox/whatsnew/qrcode-firefox-whatsnew.png') }}" width="250" height="250" alt="">
-        </div>
-      {% endif %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-md">
+    <h1 class="wnp-main-title">{{ ftl('whatsnew-s2d-download-firefox-for-your') }}</h1>
+
+    <div class="qr-wrapper">
+      <img class="qr-code" src="{{ static('img/firefox/whatsnew/qrcode-firefox-whatsnew.png') }}" width="180" height="180" alt="{{ ftl('whatsnew-s2d-qr-code-alt') }}">
     </div>
-  </section>
+
+    <ul class="mobile-download-buttons">
+      <li class="android">
+        {{ google_play_button(href=android_url, id='play-store-link') }}
+      </li>
+      <li class="ios">
+        {{ apple_app_store_button(href=ios_url, id='app-store-link') }}
+      </li>
+    </ul>
+  </div>
+</section>
 {% endblock %}
 
 {% block js %}
```

Comment:
**Suggestion:** remove `"js/base/send-to-device.es6.js"` from the `firefox_whatsnew` JS bundle?